File: spring-cloud-integrated-demo/membership-service/src/main/java/com/github/jccode/springcloud/integrateddemo/membership/service/MemberPointFlowService.java
package com.github.jccode.springcloud.integrateddemo.membership.service;

import com.github.jccode.springbootsample.core.repo.CrudMapper;
import com.github.jccode.springbootsample.core.service.CrudServiceImpl;
import com.github.jccode.springcloud.integrateddemo.membership.model.MemberPointFlow;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class MemberPointFlowService extends CrudServiceImpl<MemberPointFlow> {

    @Autowired
    public MemberPointFlowService(CrudMapper<MemberPointFlow> mapper) {
        super(mapper);
    }
}

File: spring-cloud-integrated-demo/membership-service/src/main/scala/com/github/jccode/springcloud/integrateddemo/membership/service/MemberPointFlowService.scala
package com.github.jccode.springcloud.integrateddemo.membership.service

import com.github.jccode.springbootsample.core.service.CrudServiceImpl
import com.github.jccode.springcloud.integrateddemo.membership.repo.MemberPointFlowMapper
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class MemberPointFlowService(@Autowired override val mapper: MemberPointFlowMapper) extends CrudServiceImpl(mapper) {

}
